sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/aio/_operations/_patch.py

```diff
@@ -24,27 +38,74 @@ async def upload( # pylint: disable=arguments-renamed, arguments-differ
         :type rule_id: str
         :param stream: The streamDeclaration name as defined in the Data Collection Rule.
         :type stream: str
-        :param body: An array of objects matching the schema defined by the provided stream.
-        :type body: list[any]
+        :param logs: An array of objects matching the schema defined by the provided stream.
+        :type logs: list[JSON] or IO
+        :keyword max_concurrency: Number of parallel threads to use when logs size is > 1mb.
```

Comment:
Is that 1 MB limit enforced by the service? If so, let's avoid saying "is > 1mb" and instead say something like "exceeds the limit imposed by the service".

Reply:
that's a good idea - yes it's imposed by the service